server/src/api/v1/admin/get-monthly-stats.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from "express";
+import { Sequelize } from "sequelize-typescript";
+import { wrapResponse } from "../../../functions/response-wrapper";
+import { Customer } from "../../../models/customer.models";
+import { Order } from "../../../models/order.model";
+import { Plan } from "../../../models/plan.model";
+import { User } from "../../../models/user.model";
+
+export async function getMonthlyStats(req: Request, res: Response) {
+    const customerCount = await countMonthlyEntities(Customer);
+    const userCount = await countMonthlyEntities(User);
+    const planCount = await countMonthlyEntities(Plan);
+    const orderCount = await countMonthlyEntities(Order);
+
+    const data = {
+        "customers": customerCount,
+        "users": userCount,
+        "plans": planCount,
+        "orders": orderCount
+    }
+    res.send(wrapResponse(true, data));
+}
+
+async function countMonthlyEntities(model: typeof User | typeof Customer | typeof Plan | typeof Order) {
```

Comment:
Schöner für die Lesbarkeit wäre
```suggestion
async function countMonthlyEntities(model: monthlyEntityType) {
```
und weiter unten
```ts
export type monthlyEntityType = typeof User | typeof Customer | typeof Plan | typeof Order;
```